src/main.cpp

```diff
@@ -1220,10 +1220,10 @@ bool AcceptToMemoryPoolWorker(CTxMemPool& pool, CValidationState &state, const C
 }
 
 bool AcceptToMemoryPool(CTxMemPool& pool, CValidationState &state, const CTransaction &tx, bool fLimitFree,
-                        bool* pfMissingInputs, bool fOverrideMempoolLimit, bool fRejectAbsurdFee)
+                        bool* pfMissingInputs, bool fOverrideMempoolLimit, const CAmount nAbsurdFee)
```

Comment:
I don't have background in cpp but isn't this considered best practice? (Fiddling with `nAbsurdFee` here would give `error: assignment of read-only parameter ‘nAbsurdFee’`).

Pass by reference is not mentioned in [/doc/developer-notes.md](/bitcoin/bitcoin/blob/master/doc/developer-notes.md) and I [don't like](http://stackoverflow.com/a/7701247) doing this for basic types but I am happy to change if that's something we do.